.ci.yaml

```diff
@@ -2588,7 +2588,7 @@ targets:
       dependencies: >-
         []
       tags: >
-        ["devicelab","hostonly"]
+        ["devicelab","android","mac"]
```

Comment:
We also need to update the builder name: `Mac entrypoint_dart_registrant` -> `Mac_android entrypoint_dart_registrant` to enforce an android device.

Reply:
Done, I also moved it to be next to the other Mac_android tasks.